spec/models/account_deletion_spec.rb

```diff
@@ -5,88 +5,73 @@
 require 'spec_helper'
 
 describe AccountDeletion, :type => :model do
+  let(:ad_new)    { AccountDeletion.new(person: alice.person) }
```

Comment:
I think it would be worth to write `account_deletion` out here, just use search & replace ;)